addons/l10n_ec/migrations/3.5/post-migration.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+
+from odoo import api, SUPERUSER_ID
+
+def update_withhold_income_sale_repartition_lines(env):
```

Comment:
Leave the migration script outside now.
So we can merge it and then we work on the migration script.